server/test/swagger_2/swagger_2.test.js

```diff
@@ -0,0 +1,71 @@
+const app = require('../../app')
+const supertest = require('supertest')
+const request = supertest(app)
+
+jest.setTimeout(30000)
+
+describe('POST /api/specification/swagger_2/openapi_3', () => {
+  //Swagger 2 (JSON) to OpenAPI 3 (JSON)
+  it('should convert swagger_2 (json) to openapi_3 (json) ', async () => {
+    const res = await request
+      .post('/api/specification/swagger_2/openapi_3')
+      .attach('file', 'test/swagger_2/data/swagger_2.json')
+    expect(res.status).toBe(200)
+    expect(res.text).not.toBeNull()
+    expect(res.body).toEqual({})
```

Comment:
Why is the response expected to be empty?

Reply:
The response body is empty as in jest this body is supposed to be a json body but by default the endpoint is going to return a string for all the conversions.

Reply:
Got it - can we test for the full correct response string in `res.text` then?